kqueen_ui/blueprints/manager/views.py

```diff
@@ -8,12 +8,14 @@
 from kqueen_ui.auth import generate_confirmation_token
 from kqueen_ui.blueprints.ui.utils import generate_password, sanitize_resource_metadata
 from kqueen_ui.generic_views import KQueenView
+from kqueen_ui.utils.loggers import user_prefix
 from kqueen_ui.utils.wrappers import superadmin_required
 from slugify import slugify
 
 import logging
 
-logger = logging.getLogger(__name__)
+logger = logging.getLogger('kqueen_ui')
```

Comment:
@tomkukral had some reservations regarding this setup of logger, can you refer to the KQueen logging PR for his comments?

Reply:
i already defined module naming in logger formatting (logger_config.py), it will be good for:

reuse 1 logger and does not create multiple loggers -> increase performance
all logging arch stored in one file -> increase visibility
possibility of design new logger_config for customers as 1 file-upload

Reply:
its already removed) pls take a look on commits)